installer/release.sh

```diff
@@ -102,7 +56,7 @@ case "$version" in
 *) die "InnoSetup requires a version that begins with a digit";;
 esac
 
-render_release_notes
+../render-release-notes.sh --output installer/ --css usr/share/git/
```

Comment:
If you follow my thinking about avoiding to `cd` to `build-extra/` in `render-release-notes.sh`, this should now use the `--output . --css usr/share/git/ --copy-css` options instead.